homework/02LessonTwo.js

```diff
@@ -108,7 +121,16 @@ function isDivisibleByThree(number) {
 // should return false if character is anything else
 
 function whichSpecies(character) {
+   if (character = 'scooby') {
+      return "dog";
+}  if (character = 'garfield' ){
+      return "cat";
+}  if (character = 'nemo') {
+      return "fish";
+} else {
+   return false
```

Comment:
You should be using  the "==" operator here.  "=" is used to assign variables, not compare.

Please read the following:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness